cmd/api/src/model/auth.go

```diff
@@ -565,6 +565,7 @@ type SessionAuthProvider int
 const (
 	SessionAuthProviderSecret SessionAuthProvider = 0
 	SessionAuthProviderSAML   SessionAuthProvider = 1
+	SessionAuthProviderOIDC   SessionAuthProvider = 2
```

Comment:
```suggestion
	SessionAuthProviderSecret SessionAuthProvider = iota
	SessionAuthProviderSAML
	SessionAuthProviderOIDC
```

Reply:
The `SessionAuthProvider` value is stored in the DB. If for some reason the values listed after iota were to be reorganized the representation of the data stored would change. To avoid this potential mishap we should avoid using `iota` in this case.

Reply:
Fair enough!